pref/src/main/kotlin/voice/pref/AutoSleepTimerPrefs.kt

```diff
@@ -0,0 +1,10 @@
+package voice.pref
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class AutoSleepTimerPrefs(
+  var enabled: Boolean,
+  var startTime: String,
```

Comment:
I'd use `LocalTime` here and then create a custom serializer.
You can follow the `UriSerializer`  to see that in action
```kotlin
  @Serializable
  data class SelectFolderType(
    val uri:
    @Serializable(with = UriSerializer::class)
    Uri,
    val mode: Mode,
  ) : Compose {

    enum class Mode {
      Default,
      Onboarding,
    }
  }
```